tests/time_tests/scripts/run_timetest.py

```diff
@@ -131,6 +152,14 @@ def cli_parser():
                         dest="stats_path",
                         type=Path,
                         help='path to a file to save aggregated statistics')
+    parser.add_argument('-c',
+                        dest="cache",
+                        action='store_true',
+                        help='Enable model cache usage')
+    parser.add_argument('-v',
+                        dest='vpu_mlir_compiler',
+                        action='store_true',
+                        help='Change VPUX compiler type to MLIR. Default compiler is MCM')
```

Comment:
This option looks extremely plugin specific. Even for VPU plugin itself, the default can be changed (and it will). Do we really need such option in the generic tool?

@ArtemySkrebkov-intel @rzubarev What's your opinion?

Reply:
this option was added on request @olegmorenov

Reply:
I re-call we agreed to implement a config file. It will allow us passing a config option to enable MLIR compiler.

The config file is mentioned in https://jira.devtools.intel.com/browse/CVS-68822

Reply:
Moreover, the default compiler in VPU plugin will be changed soon, so this option won't have any effect.